polaris/ocean/ocean.cfg

```diff
@@ -3,6 +3,16 @@
 
 # Options related the ocean component
 [ocean]
+
+# Tolerance for mass conservation, normalized by total mass
+mass_conservation_tolerance = 1e-8
+
+# Tolerance for salt conservation, normalized by total salt
+salt_conservation_tolerance = 1e-8
+
+# Tolerance for thermal energy conservation, normalized by total energy
+energy_conservation_tolerance = 1e-8
+
```

Comment:
I'm somewhat unsure as to whether this or the ocean model section would be a better choice. The latter would imply that we have different tolerances for different models.